<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="HP" failures="1" tests="2" name="Default test(failed)" time="0.151" errors="0" timestamp="2020-09-16T21:39:12 IST">
  <testcase classname="Demo.TestNGRetryFailedDemo" name="test3" time="0.028">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Demo.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="Demo.TestNGRetryFailedDemo" name="test2" time="0.004"/>
  <testcase classname="Demo.TestNGRetryFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
